pkg/util/clusterdata/clusterdata.go

```diff
@@ -108,9 +123,10 @@ out:
 		case f := <-callbacks:
 			f()
 		case <-errors:
-			// Ignore errors. We log them in each task locally
-			continue
+			// No need to log errors. We log them in each task locally
+			e.m.EmitGauge("enricher.tasks.errors", 1, nil)
 		case <-ctx.Done():
+			e.m.EmitGauge("enricher.timeouts", 1, nil)
```

Comment:
I'm not sure if we need more details in metrics. For example, add dimension with task type?

Reply:
I feel like I don't know enough about this code to be able to make a suggestion either way.